src/api/app/controllers/webui/staging_workflows_controller.rb

```diff
@@ -40,6 +39,26 @@ def show
     @project = @staging_workflow.project
   end
 
+  def destroy
+    @staging_workflow = StagingWorkflow.find_by(id: params[:id])
+    @project = @staging_workflow.project
+
+    #TODO: authorization
+    
+    if @staging_workflow.staging_projects
```

Comment:
This block shouldn't be needed because the [staged_request association](https://github.com/openSUSE/open-build-service/blob/staging-workflow/src/api/app/models/staging_workflow.rb#L10) could have a `dependent: nullify`.

Regarding deleting the staging projects , if you look into the [staging workflow model](https://github.com/openSUSE/open-build-service/blob/staging-workflow/src/api/app/models/staging_workflow.rb#L8) we also have a `dependent: nullify`. So I'm not sure all if we want to delete them.

Reply:
@DavidKang Allright, i will work on it :)

Reply:
done :dancer: